### PR TITLE
Document that `skip_collapse` turns on paired-end alignment.

### PR DESCRIPTION
**Pull request by charles-plessy**
Following issue #652, I would like to propose to explicitly mention that `skip_collapse` turns on paired-end alignment, so that a keyword search in the _parameters_ page of the documentation will reveal the information.

(PS: sorry for the newline change at the end of the patch; it seems that GitHub added it automatically).

### CODE REVIEW

**Review by apeltzer — CHANGES_REQUESTED** [review]
please also add a commen to changelog.md to mention this 👍🏼 Thanks for making it :-)

File: nextflow_schema.json [hunk 1]
```diff
@@ -472,7 +472,7 @@
                 },
                 "skip_collapse": {
                     "type": "boolean",
-                    "description": "Skip of merging forward and reverse reads together. Only applicable for paired-end libraries.",
+                    "description": "Skip of merging forward and reverse reads together and turns on paired-end alignment. Only applicable for paired-end libraries.",
```
Comment (jfy133) [inline nextflow_schema.json:475]:
```suggestion
                    "description": "Skip of merging forward and reverse reads together and turns on paired-end alignment for downstream mapping. Only applicable for paired-end libraries.",
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -472,7 +472,7 @@
...
                     "fa_icon": "fas fa-fast-forward",
                     "help_text": "Turns off the paired-end read merging.\n\nFor example\n\n```bash\n--skip_collapse  --input '*_{R1,R2}_*.fastq'\n```\n\nIt is important to use the paired-end wildcard globbing as `--skip_collapse` can only be used on paired-end data!\n\n:warning: If you run this and also with `--clip_readlength` set to something (as is by default), you may end up removing single reads from either the pair1 or pair2 file. These will be NOT be mapped when aligning with either `bwa` or `bowtie`, as both can only accept one (forward) or two (forward and reverse) FASTQs as input.\n\n> Modifies AdapterRemoval parameter: `--collapse`"
```
Comment (jfy133) [inline nextflow_schema.json:477]:
```suggestion
                    "help_text": "Turns off the paired-end read merging.\n\nFor example\n\n```bash\n--skip_collapse  --input '*_{R1,R2}_*.fastq'\n```\n\nIt is important to use the paired-end wildcard globbing as `--skip_collapse` can only be used on paired-end data!\n\n:warning: If you run this and also with `--clip_readlength` set to something (as is by default), you may end up removing single reads from either the pair1 or pair2 file. These will be NOT be mapped when aligning with either `bwa` or `bowtie`, as both can only accept one (forward) or two (forward and reverse) FASTQs as input.\n\nAlso note that supplying this flag will also then cause downstream mapping steps to run in paired-end mode. This maybe more suitable for modern data, or when you want to utilise mate-pair spatial information.\n\n> Modifies AdapterRemoval parameter: `--collapse`"
```

**Review by jfy133 — CHANGES_REQUESTED** [review]
@charles-plessy thanks for the PR (and sorry for my slow follow up)! I've made a commit suggestion and also expanded the PR so the information is also described in the 'detailed' help information (not just the short one). Please commit the suggestions if you're happy with them.

As of course, as @apeltzer says, please update the changelog. You're also welcome to add yourself ot the 'Additional Contributors' of the README if you wish, just follow the format as for those already there.

**Comment by jfy133** [comment]
@apeltzer I updated the changelog (to get this in), if your'e happy, please approve and merge.

File: README.md [hunk 3]
```diff
@@ -159,6 +159,7 @@ of this pipeline:
 Those who have provided conceptual guidance, suggestions, bug reports etc.
 
 * Arielle Munters
+* [Charles Plessey](https://github.com/charles-plessy)
```
Comment (charles-plessy) [inline README.md:162]: Sorry there is a typo in my name... It is _Plessy_.

Comment (apeltzer) [inline README.md:162]:
```suggestion
* [Charles Plessy](https://github.com/charles-plessy)
```

Comment (apeltzer) [reply README.md:162]: Fixed it!

**Review by jfy133 — APPROVED** [review]

**Pull request merged**